go.mod

```diff
@@ -208,3 +208,5 @@ replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.3
 
 // Fix CVE-2021-41190
 replace github.com/opencontainers/image-spec => github.com/opencontainers/image-spec v1.0.2
+
+replace github.com/fluxcd/source-controller/api => github.com/fluxcd/source-controller/api v0.21.3-0.20220216150237-6804b676eb32
```

Comment:
I think we should delete this and set the api version to v.22.0 in the first section.